src/PKSim.Core/Model/IndividualSimulation.cs

```diff
@@ -1,69 +1,26 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using OSPSuite.Utility.Collections;
 using OSPSuite.Utility.Extensions;
 using OSPSuite.Utility.Visitor;
 using OSPSuite.Core.Domain;
 using OSPSuite.Core.Domain.Data;
 using OSPSuite.Core.Domain.Services;
 using PKSim.Core.Chart;
+using PKSim.Core.Model.Extensions;
 
 namespace PKSim.Core.Model
 {
    public class IndividualSimulation : Simulation
    {
       private DataRepository _dataRepository;
-      private readonly ICache<string, CompoundPK> _compoundPKCache;
 
-      public IndividualSimulation()
+      public IndividualSimulation() : base()
```

Comment:
calling base() is not required. Remove

Reply:
Why not? There is some code in the base constructor that needs to be executed still. More specifically 
``_compoundPKCache = new Cache<string, CompoundPK>(x => x.CompoundName, x => new CompoundPK());``

Reply:
Because the base() constructor is always called....or am I missing something?